frontend/test/pytest/test_qnode.py

```diff
@@ -72,7 +72,7 @@ def g(x: float):
 def test_unsupported_device():
     """Test unsupported device."""
 
-    @qml.qnode(qml.device("default.qubit", wires=2))
+    @qml.qnode(device=qml.device("default.qubit", wires=2))
```

Comment:
how come this changed?

Reply:
Not necessary, I had an issue with device and it was because of new device do not have short name